src/langsmith/evaluation-quickstart.mdx

```diff
@@ -3,131 +3,278 @@ title: Evaluation quickstart
 </Tip>
 
-## LangSmith SDK
+## 1. Install dependencies
 
-### 1. Install dependencies
+In your terminal, create a directory for your project and install the dependencies in your environment:
 
 <CodeGroup>
 
 ```bash Python
+mkdir ls-evaluation-quickstart && cd ls-evaluation-quickstart
+python -m venv .venv && source .venv/bin/activate
+python -m pip install --upgrade pip
 pip install -U langsmith openevals openai
 ```
 
 ```bash TypeScript
+mkdir ls-evaluation-quickstart-ts && cd ls-evaluation-quickstart-ts
+npm init -y
 npm install langsmith openevals openai
+npx tsc --init
 ```
 
 </CodeGroup>
 
 <Info>
-If you are using `yarn` as your package manager, you will also need to manually install `@langchain/core` as a peer dependency of `openevals`. This is not required for LangSmith evals in general - you may define evaluators [using arbitrary custom code](/langsmith/code-evaluator).
+If you are using `yarn` as your package manager, you will also need to manually install `@langchain/core` as a peer dependency of `openevals`. This is not required for LangSmith evals in general, you may define evaluators [using arbitrary custom code](/langsmith/code-evaluator).
 </Info>
 
-### 2. Create a LangSmith API key
-
-To create an API key, head to the [Settings page](https://smith.langchain.com/settings). Then click **+ API Key.**
+## 2. Set up environment variables
 
-### 3. Set up environment variables
+Set the following environment variables:
 
-This guide uses OpenAI, but you can adapt it to use any LLM provider.
-
-If you're using Anthropic, use the [Anthropic wrapper](/langsmith/annotate-code#wrap-the-anthropic-client-python-only) to trace your calls. For other providers, use [the traceable wrapper](/langsmith/annotate-code#use-%40traceable-%2F-traceable).
+- `LANGSMITH_TRACING`
+- `LANGSMITH_API_KEY`
+- `OPENAI_API_KEY` (or your LLM provider's API key)
+- (optional) `LANGSMITH_WORKSPACE_ID`: If your LangSmith API is linked to multiple [workspaces](/langsmith/administration-overview#workspaces), set this variable to specify which workspace to use.
 
-```shell
+``` bash
 export LANGSMITH_TRACING=true
-export LANGSMITH_API_KEY="<your-langchain-api-key>"
+export LANGSMITH_API_KEY="<your-langsmith-api-key>"
 export OPENAI_API_KEY="<your-openai-api-key>"
+export LANGSMITH_WORKSPACE_ID="<your-workspace-id>"
 ```
 
-### 4. Create a dataset
+If you're using Anthropic, use the [Anthropic wrapper](/langsmith/annotate-code#wrap-the-anthropic-client-python-only) to trace your calls. For other providers, use [the traceable wrapper](/langsmith/annotate-code#use-%40traceable-%2F-traceable).
 
-Next, define example input and reference output pairs that you'll use to evaluate your app:
+## 3. Create a dataset
+
+1. Create a `dataset` file and add the following code, which will:
```

Comment:
was confused by "create a dataset file", at first assumed we were creating a file without our dataset examples